BootstrapScriptWithToken/bootstrap.py

```diff
@@ -189,6 +210,13 @@ def __init__( self ):
       sys.exit( "error: address to CVP missing" )
    if enrollmentToken == "":
       sys.exit( "error: enrollment token missing" )
+   if currentTimeDate == "":
+      sys.exit("error: Current Time and Date missing")
+   elif currentTimeDate == "ntp" or currentTimeDate == "NTP":
+      setNTPsync(ntp_server = "time.google.com")
+      setNTPsync(ntp_server = "pool.ntp.org")
```

Comment:
I don't think we should assume that either of the servers would necessarily be reachable from the device. We should also think about the case where customer doesn't want to setup NTP, we should allow the user to make that choice.

Reply:
dns server ip local or otherwise is given in dhcp pools by practice. i can hard-code the physical ntp ip addresses as well.

Reply:
I think the only change we should keep for all of our customers is the NTP part of the change. We should take a list of NTP servers as an argument from the customer in the script, and use those to sync time. We should remove other arguments. If you can update the change, that'd be great. If not, I will try to get to it as soon as I can. Thanks